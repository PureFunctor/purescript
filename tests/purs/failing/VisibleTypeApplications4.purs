-- @shouldFailWith CannotSkipTypeApplication
module Main where

class Foo @foo where
  foo :: String

foo' = foo @_
