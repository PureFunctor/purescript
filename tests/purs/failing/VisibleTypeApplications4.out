Error found:
in module [33mMain[0m
at tests/purs/failing/VisibleTypeApplications4.purs:7:8 - 7:14 (line 7, column 8 - line 7, column 14)

  A type variable:
  [33m     [0m
  [33m  foo[0m
  [33m     [0m
  in the type:
  [33m                                [0m
  [33m  forall @foo. Foo foo => String[0m
  [33m                                [0m
  cannot be skipped during type application.

while inferring the type of [33mfoo[0m
in value declaration [33mfoo'[0m

See https://github.com/purescript/documentation/blob/master/errors/CannotSkipTypeApplication.md for more information,
or to contribute content related to this error.

